src/MoBi.R/RRegister.cs

```diff
@@ -30,6 +37,7 @@ public override void RegisterInContainer(IContainer container)
 
       private static void registerMinimalTypes(IContainer container)
       {
+         container.Register<Core.Domain.Services.ISimulationFactory, Core.Domain.Services.SimulationFactory>();
```

Comment:
That should not be needed. It should be registered by `CoreRegister` try to remove and see if everything still works